test/e2e/resources/feature_group.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: FeatureGroup
+metadata:
+  name: $FEATURE_GROUP_NAME
+spec:
+  eventTimeFeatureName: EventTime
+  featureDefinitions:
+    - featureName: TransactionID
+      featureType: Integral
+    - featureName: EventTime
+      featureType: Fractional
+  featureGroupName: $FEATURE_GROUP_NAME
+  recordIdentifierFeatureName: TransactionID
+  offlineStoreConfig:
+    s3StorageConfig:
+      s3URI: s3://$SAGEMAKER_DATA_BUCKET/feature-group-data
```

Comment:
nit: all the test data is under sagemaker directory, can we follow same pattern here